integrations/operator/crdgen/cmd/protoc-gen-crd-docs/debug.go

```diff
@@ -0,0 +1,58 @@
+//go:build debug
+
+/*
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package main
+
+import (
+	"os"
+
+	"github.com/gravitational/trace"
+	log "github.com/sirupsen/logrus"
```

Comment:
Since the original scope of this PR was to make the minimum set of changes required to format CRD data into documentation pages, would it make sense to use `slog` in a separate PR? The original CRD generator plugin uses `logrus`, and it doesn't seem like `slog` is a drop-in replacement.